### DISCUSSION TITLE
"Billing" type user

### DISCUSSION THREAD

**Discussion by eastofwestern** [vercel/community#145] [Idea]
Hello,

I have a question about setting up client's on a pro team.  Our specific situation is building client sites and maintaining the codebase in our agency GIT repository's.  Our normal workflow is building out the site on a free plan during development and then when ready to go live we upgrade to a TEAM account.  Ideally we want to stay a member of the team but would love to have the client also join to be responsible for billing.  The client would have no role other than being responsible for credit card details.  Adding them to the account to take care of billing jumps to $40 a month.  Wondering if there is a scenario where a "billing" type user with very limited access could be added?  For smaller projects the $20 a month price point is much easier to sell into clients then the $40 a month.  Especially clients coming from the GoDaddy type shared hosting world.

Curious on any feedback and thank you for an amazing service.

Neal

**Comment by Pieparker** [comment]
Thanks for the suggestion, Neal! I've got this logged internally. We'll be sure to let you know if anything arises from this.

**Comment by malcolm-t3** [comment]
I agree with this feature idea! I think there should be a way to have a non-member (free) user type that just manages billing, admin, operational stuff without the need to actually use the product.

**Comment by kvilesjo** [comment]
@Pieparker any update on this? It would be great to get some positive feedback at this topic, so we can plan for at broader adoption of Vercel.

**Reply by Pieparker** [reply]
Nothing to report as far as our roadmap goes, @kvilesjo. My advice for the time-being would be to ensure that you're upvoting these topics! If you're currently in discussions with our sales team regarding an Enterprise agreement, please ensure that you share this requirement directly with them.

**Comment by estellarspace** [comment]
Wanted to chime in, this would help our initiative as well!

**Comment by Danm72** [comment]
Would also love this.

**Comment by leerob** [comment]
You can now add a free team seat for managing billing on Pro: https://vercel.com/changelog/billing-role-on-pro
